### ISSUE TITLE
Adding other evaluations to student application view [2h]

### ISSUE THREAD

**Issue by BurakCemOztarak** [BurakCemOztarak/FH_Final_Project#18]
Issue
----

- https://qa-schuler-program.pantheonsite.io/admin/student-application

Adding "Outside Evaluators" will also affect `/admin/student-application` table and currently, we have a column that shows Teacher evaluations only. Therefore, "Teacher" column should be modified and also be able to show the situation of outside evaluations. 

<img width="1224" alt="Screen Shot 2020-12-03 at 1 43 19 PM" src="https://user-images.githubusercontent.com/73833912/101003687-67900380-3571-11eb-90cf-f91245371a9d.png">

- Basically, teacher evaluation column should be able to show all missing evaluations and the title of the column should be changed accordingly.

******
To do
----

- Change the title of the column. `Teacher` to `Evaluation`

- "Outside Evaluation" is needed to be added under the same column.

<img width="768" alt="Screenshot 2020-12-03 at 13 22 53" src="https://user-images.githubusercontent.com/34378755/101017669-b1292000-356a-11eb-8fe1-0e05c1f6f24c.png">
